api_client/src/lib.rs

```diff
@@ -141,18 +141,19 @@ fn parse_http_response(socket: &mut dyn Read) -> Result<Option<String>, Error> {
     }
 }
 
-pub fn simple_api_command_with_fds<T: Read + Write + ScmSocket>(
+pub fn simple_api_command_with_fds_and_prefix<T: Read + Write + ScmSocket>(
```

Comment:
I think it would be cleaner to have the API take in the full API command e.g. `vm.boot` and then it can be used for vmm commands.

Reply:
@rbradford - Good call! Branch updated.